index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Service Worker Sample Application</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+
+  <header class="header">
+    <div class="container">
+      <div class="header__layout">
+        <h1 class="header__logo-wrapper">
+          <a href="#" class="logo">Logo</a>
+        </h1>
+        <nav>
+          <ul class="menu">
+            <li class="menu__item"><a href="#" class="menu__link">Home</a></li>
+            <li class="menu__item"><a href="#" class="menu__link">About</a></li>
+            <li class="menu__item"><a href="#" class="menu__link">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </header>
+
+  <section class='section--services'>
```

Comment:
please use BEM block naming here, if modifier is needed user after block.<br/>
example: section class='section   section--services'